app/build.gradle

```diff
@@ -58,25 +58,25 @@ dependencies {
     kapt 'androidx.lifecycle:lifecycle-compiler:2.0.0'
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.arch.core:core-testing:2.0.1'
-    testImplementation 'android.arch.core:core-testing:1.0.0-alpha3'
```

Comment:
testImplementation 'android.arch.core:core-testing:1.0.0-alpha3'

mixing core components from different packages causes unexpected errors.. So I changed this to jetpack androidx arch components